manifests/nodejs.yml

```
@@ -221,7 +221,27 @@ tests/:
         test_uri.py:
           TestURI: missing_feature
     rasp/:
-      test_lfi.py: missing_feature
+      test_lfi.py:
+        Test_Lfi_BodyJson:
+          '*': *ref_5_24_0
+          nextjs: missing_feature
+        Test_Lfi_BodyUrlEncoded:
+          '*': *ref_5_24_0
+          nextjs: missing_feature
+        Test_Lfi_BodyXml: missing_feature
+        Test_Lfi_Capability: *ref_5_24_0
+        Test_Lfi_Mandatory_SpanTags: *ref_5_24_0
+        Test_Lfi_Optional_SpanTags: *ref_5_24_0
+        Test_Lfi_RC_CustomAction: missing_feature
```

why does this one not work tho ?

---

https://github.com/DataDog/system-tests/pull/3226

---

to be modified when the above PR is merged
```suggestion
        Test_Lfi_RC_CustomAction: *ref_5_24_0
```